allauth_2fa/middleware.py

```diff
@@ -95,6 +96,15 @@ def process_view(self, request, view_func, view_args, view_kwargs):
             return
 
         # If the user is on one of the allowed pages, do nothing.
+        for urlname in self.allowed_pages:
+            try:
+                if request.path == reverse(urlname):
+                    return
+            except NoReverseMatch:
+                # The developer may have misconfigured the list of allowed pages.
+                # Let's not outright crash at that point, but inform the developer about their mishap.
+                warnings.warn('NoReverseMatch for %s while checking for pages allowed without 2FA' % urlname)
+
         if request.path in map(reverse, self.allowed_pages):
```

Comment:
I believe this code is now duplicative to the newly added for-loop.

Reply:
My bad.